updates/julian_2.sql

```diff
@@ -9,11 +9,16 @@ VALUES ('einzelbeleg','NewAreal','combined',1,'einzelbeleg_hatareal','ArealID;Ar
 
 ALTER TABLE `einzelbeleg_hatareal` ADD COLUMN `ArealTypID` int DEFAULT -1;
 
+ALTER TABLE einzelbeleg_hatareal ADD CONSTRAINT einzelbeleg_hatareal_ArealTypID FOREIGN KEY (ArealTypID) REFERENCES selektion_arealtyp(ID);
```

Comment:
This foreign key cannot be added before the new table selektion_arealtyp is created, so it should be moved behind the corresponding CREATE TABLE statement.

Also, please keep a certain normalized syntax at least within the same SQL file => in all other statements, table + column names are wrapped with backticks.

Reply:
fixed!